lib/seattleflu/db/cli/command/etl/enrollments.py

```diff
@@ -103,6 +115,57 @@ def etl_enrollments(*, action: str):
                     site_id         = site.id,
                     details         = encounter_details(enrollment.document))
 
+                # Find existing collected samples, or create skeletal sample
+                # records containing just the collection barcode linked back to
+                # this encounter.  Sample manifests generated by the processing
+                # lab will usually be loaded later and fill in the rest of the
+                # sample record.
+                for sample in enrollment.document["sampleCodes"]:
+                    barcode = sample.get("code")
+
+                    if not barcode:
+                        LOG.warning(f"Skipping collected sample with no barcode")
+                        continue
+
+                    # XXX TODO: Stop hardcoding this and handle other types.
+                    # ScannedSelfSwab and ManualSelfSwabbed are kit barcodes,
+                    # not collection barcodes.  TestStrip is an identifier
+                    # UUID, not barcode.
+                    #   - trs, 17 May 2019
+                    if sample["type"] != "ClinicSwab":
+                        LOG.warning(f"Skipping collected sample with unknown type {sample['type']}")
+                        continue
+
+                    LOG.debug(f"Looking up collected sample code «barcode»")
```

Comment:
it looks like you forgot to wrap barcode here in curly brackets

Reply:
So I did! I pushed a fixup commit addressing that.